contracts/POA/Conquest.sol

```diff
@@ -0,0 +1,175 @@
+pragma solidity ^0.6.8;
+pragma experimental ABIEncoderV2;
+
+import "../utils/Ownable.sol";
+import "../interfaces/ISkyweaverAssets.sol";
+import "multi-token-standard/contracts/utils/SafeMath.sol";
+import "multi-token-standard/contracts/interfaces/IERC165.sol";
+import "multi-token-standard/contracts/interfaces/IERC1155.sol";
+import "multi-token-standard/contracts/interfaces/IERC1155TokenReceiver.sol";
+
+/**
+ * Contract used on POA to internally keep track of players participation
+ * in Conquest. 
+ */
+contract Conquest is IERC1155TokenReceiver, Ownable {
+  using SafeMath for uint256;
+
+  /***********************************|
+  |             Variables             |
+  |__________________________________*/
+
+  uint256 constant public TIME_BETWEEN_CONQUESTS = 5 minutes;  // Seconds that must elapse between two conquest
+  uint256 constant public MAX_REWARD_AMOUNT = 200;             // Maximum number of cards one can win from a conquest (2 decimals)
+
+  ISkyweaverAssets immutable public skyweaverAssets; // ERC-1155 Skyweaver assets contract
+  IERC1155 immutable public conquestEntryAddress;    // ERC-1155 address containing conquest entry
+  uint256 immutable public conquestEntryID;          // Conquest entry token id
+
+  mapping(address => bool) public isActiveConquest;    // Whether a given player is currently in a conquest
+  mapping(address => uint256) public nextConquestTime; // Time when the last conquest started
```

Comment:
I think that the comment should state `time when the next conquest can start` or something similar

Reply:
Good catch